### ISSUE TITLE
Deezer / Spotify / Napster

### ISSUE THREAD

**Issue by r3b00tl00p** [junaid33/opensource.builders#317]
Music streaming platforms

**Comment by evanspy1** [comment]
Tidal is another closed-source one, but I am not aware of any FOSS music streaming platforms...

**Comment by jcklpe** [comment]
Closest thing I can think of is https://resonate.is/  
  
It's a music streaming coop. It looks at least part of their tech is open source though they aren't primarily focusing on that so IDK where the repo is for it: https://resonate.is/going-beyond-the-app-store-model/

**Comment by evanspy1** [comment]
Yeah, and the song availability seems to be kinda "meh".

I guess there are foss mp3 players/clients (jellyfin, mstream, etc.) but those aren't really direct competition because those are all bring-your-own-music.

**Comment by jcklpe** [comment]
Yeah availability is going to be a problem almost no matter what as the actual songs and intellectual property aren't open source, kind of by nature. Closet thing would perhaps be for someone to build some kind of streaming service that utilized bittorrent on the backend, sorta like how Popcorn Time used to work

**Comment by jcklpe** [comment]
Looks like there used to be a thing like that in 2015 called Aurous but it was shut down by the RIAA. If it was open source I don't understand how it was shut down though. I'm doing more googling on the matter.

**Comment by jcklpe** [comment]
This could have potential: https://github.com/IrosTheBeggar/mStream

**Comment by zamgreg** [comment]
This? Federated (using ActivityPub), btw
https://dev.funkwhale.audio/funkwhale

Also know about this (but never used it)
https://github.com/airsonic/airsonic

**Comment by zamgreg** [comment]
Another one powered by ActivityPub:
https://reel2bits.org

**Comment by jcklpe** [comment]
Thanks @zamgreg  these are great. Really interesting seeing a Fediverse solution to this stuff.
